README.md

```diff
@@ -77,6 +77,11 @@ deploy-stack dev
 * To bootstrap the project: `build-stack`
 * To run the project: `start-stack`
 * To stop the project: `stop-stack`
+* To run a command inside a docker container: `ma-exec <CONTAINER_NAME> <COMMAND>`. Below is a list of useful container specific commands.
+  * Run API data validation tests: `ma-exec api yarn test`
+  * Use prettier to format frontend (also works for `api`) files: `ma-exec frontend format`
+  * Run eslint for frontend (also works for `api`): `ma-exec frontend lint`
```

Comment:
Should be `ma-exec frontend yarn lint`

Reply:
A bit unclear what's the different between `yarn lint` and `yarn format`. Maybe briefly describe it or link to further reading?

Reply:
Thank you for spotting these @nanjiangshu! Fixed now.

Reply:
> A bit unclear what's the different between `yarn lint` and `yarn format`. Maybe briefly describe it or link to further reading?

We have some more information about lint and prettier [here](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/CONTRIBUTING.md#prettier). I would prefer to keep this section (list of commands) as light as possible if everyone is ok with that.

Reply:
It makes sense to keep the content here as brief as possible. A link will be good enough.

Reply:
Have added the links now for prettier and eslint.